### DISCUSSION TITLE
Add Camera

### DISCUSSION THREAD

**Discussion by AlbertLCS** [netalertx/NetAlertX#95] [Ideas]
https://github.com/jokob-sk/Pi.Alert/blob/97305026a78e11621c705faca0a9a04af14b74e3/front/php/server/devices.php#L794

could you add camera?

**Comment by jokob-sk** [comment]
Hi!

Sure!

Here you go: https://github.com/jokob-sk/Pi.Alert/commit/dc5deaab424bbb2946f0042bc995a9a0fea55592

I will release a new image probably tomorrow, if impatient, try downloading the dev image instead ;) : https://registry.hub.docker.com/r/jokobsk/pi.alert_dev
